helm/cluster-autoscaler-app/values.schema.json

```diff
@@ -39,13 +39,13 @@
                     "type": "object",
                     "properties": {
                         "http": {
-                            "type": ["string","null"]
+                            "type": "string"
```

Comment:
this can definitely be null so I guess you need the array notation

Reply:
I see you changed the default to empty string, I guess that works as well

Reply:
Yes, I changed the default.

Just leaving the value empty (which intends to be `null`) probably works, but I feel like this can also cause some difficult to track down issues.

In this particular use case, empty string is being handled the same as `null`, so we should be fine with just `string`.